api/appendices/resource_attributes.adoc

```diff
@@ -74,6 +75,33 @@ POST /api/rates
 
 ___
 
+[[groups]]
+=== Group Attributes
+
+----
+POST /api/groups
+----
+
+==== Required attributes:
+
+[cols="2,1,3",options="header",width="100%"]
+|=====================
+| Attribute | Type | Description
+| description | string | Description
+| role | hash | Role the group is assigned to, requires an *href*, *id* or *name* attribute in the hash
```

Comment:
Thanks Tim, I've updated and pushed up the others. For this (and all *hash* types here), Object is not necessarily always true, as our own hashes are for referencing only. We do mention the word *struct* for these in the Overview/Json Convention file.  We can update all to struct but I'm not sure   it's be clearer.